packages/browser-tests/src/api/shared.ts

```diff
@@ -0,0 +1,305 @@
+  waku: LightNode,
+  contentTopic: string,
+  payload?: Uint8Array,
+  options?: {
+    clusterId?: number;
+    shard?: number;
+  }
+): Promise<SDKProtocolResult> {
+  if (!waku) {
+    throw new Error("Waku node not found");
+  }
+  // await waku.waitForPeers(["lightpush"]);
+  const encoder = createEncoder({
+    contentTopic,
+    pubsubTopicShardInfo: {
+      clusterId: options?.clusterId ?? 1,
+      shard: options?.shard ?? 1
+    }
+  });
+
+  const result = await waku.lightPush.send(encoder, {
+    payload: payload ?? new Uint8Array()
+  });
+  return result;
+}
+
+/**
+ * Creates and initializes a Waku node
+ * Checks if a node is already running in window and stops it if it exists
+ */
+export async function createWakuNode(
+  options: CreateNodeOptions
+): Promise<{ success: boolean; error?: string }> {
+  // Check if we're in a browser environment and a node already exists
+  if (typeof window === "undefined") {
+    return { success: false, error: "No window found" };
+  }
+
+  try {
+    if ((window as any).waku) {
+      await (window as any).waku.stop();
+    }
+    (window as any).waku = await createLightNode(options);
+    return { success: true };
+  } catch (error: any) {
+    return { success: false, error: error.message };
+  }
+}
+
+export async function startNode(): Promise<{
+  success: boolean;
+  error?: string;
+}> {
+  if (typeof window !== "undefined" && (window as any).waku) {
+    try {
+      await (window as any).waku.start();
+      return { success: true };
+    } catch (error: any) {
+      // Silently continue if there's an error starting the node
+      return { success: false, error: error.message };
+    }
+  }
+  return { success: false, error: "Waku node not found in window" };
+}
+
+export async function stopNode(): Promise<{
+  success: boolean;
+  error?: string;
+}> {
+  if (typeof window !== "undefined" && (window as any).waku) {
+    await (window as any).waku.stop();
+    return { success: true };
+  }
+  return { success: false, error: "Waku node not found in window" };
+}
+
+export async function dialPeers(
+  waku: LightNode,
+  peers: string[]
+): Promise<{
+  total: number;
+  errors: string[];
+}> {
+  const total = peers.length;
+  const errors: string[] = [];
+
+  await Promise.allSettled(
+    peers.map((peer) =>
+      waku.dial(peer).catch((error: any) => {
+        errors.push(error.message);
+      })
+    )
+  );
+
+  return { total, errors };
+}
+
+export async function subscribe(
+  waku: LightNode,
+  contentTopic: string,
+  options?: {
+    clusterId?: number;
+    shard?: number;
+  },
+  callback?: (message: DecodedMessage) => void
+): Promise<SubscribeResult> {
+  const clusterId = options?.clusterId ?? 42;
+  const shard = options?.shard ?? 0;
+
+  // eslint-disable-next-line no-console
+  console.log(
+    `Creating decoder for content topic ${contentTopic} with clusterId=${clusterId}, shard=${shard}`
+  );
+
+  // Construct the pubsub topic
```

Comment:
in this file and others a lot of comments like this are not needed and just create noise

I understand that these are most likely added by AI,
but if we need a comment - let's better try to write a self-documenting code - https://jawlon.medium.com/clean-code-writing-readable-and-maintainable-code-946a5b28a1d9

🙏  @waku-org/js-waku